src/app/shared/formly/types/phone-field/phone-field.component.ts

```diff
@@ -0,0 +1,13 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { FieldType } from '@ngx-formly/core';
+
+@Component({
+  selector: 'ish-phone-field',
+  templateUrl: './phone-field.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class PhoneFieldComponent extends FieldType {
+  pattern: RegExp = /^(?:\+|00)[1-9]\d{0,2}(?: |\-)|((?: |-)[0-9]{3}(?: |-)[0-9]{4})|([0-9]{7,14})$/;
```

Comment:
Something is wrong with this RegEx (see comment browser error.).

Reply:
i think, it would be better to use a special phone number validator instead using the pattern here, because the field is marked red in case of an error but there is no error message.
You could add a new validator in the SpecialValidators class and assign it per default as it is in the email-field.component